### DISCUSSION TITLE
Error Downloading Video

### DISCUSSION THREAD

**Discussion by ArsenioLupin** [aclap-dev/video-downloadhelper#974] [Help and Q&A]
### Your question

Hi,
I just purchased the license for premium download.

While downloading videos from a course some are ok and same are corrupted (same account, same course, only different lessons)
I tried download different times the same video but it's always corrupted

![Image](https://github.com/aclap-dev/video-downloadhelper/assets/25608107/ff34bf4a-e76f-4694-99da-521e25e7cce2)

My configuration is this

version: 9.0.2.12
target: google
channel: stable
lang: de
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"xxxxxxxxxxxxxxxx"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "theme": "system",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 10,
    "successfull_dl": 81
}

Any ideas?
Thanks

### Addon & CoApp version and details

My configuration is this

version: 9.0.2.12
target: google
channel: stable
lang: de
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"xxxxxxxxxxxxxxxx"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "theme": "system",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 10,
    "successfull_dl": 81
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

The image you show is a classic, textbook case of what you get when you try to download content that is protected by DRM.  There's lots of discussion of DRM in this forum.  Find some & read it.  There's even reports of some of these education web sites having some content protected by DRM & other content not protected by DRM.  The DRM stuff is never going to download.  The stated purpose of DRM is to prevent the protected content against download.

Basically, you're out of luck.  VDH has never, does not now, nor will ever break DRM protection.  The VDH guys have repeatedly stated that in this forum & in our previous forum on Google Groups.  You are stuck.

**Comment by ArsenioLupin** [comment]
Hi,
yeah but I was wondering why in the same course some lessons can be downloaded and some not...  so in the same course some video are DRM protected and some not
PS: by the way I paid for the course

**Reply by GrampaWildWilly** [reply]
This is exactly what a number of our fellow users have said.  They pay for a course, the first lesson downloads in VDH just fine, but the rest of the course is behind DRM protection.  What are these people afraid of?  You paid for the course.  In the old days, you bought the text book & could read it whenever & wherever.  This should still be the case.  I do so hate DRM.
